presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -215,6 +215,12 @@ public void printFinalInfo()
             if (stats.getSpilledBytes() > 0) {
                 reprintLine("Spilled: " + formatDataSize(bytes(stats.getSpilledBytes()), true));
             }
+
+            if (stats.getRuntimeStats() != null) {
+                stats.getRuntimeStats().getMetrics().values().forEach(
+                        metric -> reprintLine(
```

Comment:
Do we need to worry that there might be too many runtime stats get printed out in CLI?

Reply:
This is only printed out in debug mode which should be fine.